.github/workflows/ci-ruby.yml

```diff
@@ -218,6 +194,8 @@ jobs:
           key: ${{ runner.os }}-bazel-ruby-unit-test-${{ matrix.ruby }}-${{ hashFiles('**/BUILD.bazel') }}
           restore-keys: |
             ${{ runner.os }}-bazel-ruby-unit-test-${{ matrix.ruby }}-
+      - name: Patch WORKSPACE to select Ruby ${{ matrix.ruby }}
+        run: sed -i -E "s/(rb_download\(version = \")(.+)(\"\))/\1${{ matrix.ruby }}\3/" WORKSPACE
```

Comment:
We can't set this by ENV or something "easier?"

Reply:
I don't think we can, but I would like to make it "proper" later and add support for multiple toolchains. I think this is hacky but works for now.

Reply:
@shs96c Do you know how I can define multiple `rb_download()` for different versions of Ruby in `WORKSPACE` and then select them on CI?

Reply:
The simplest thing would to `load` the ruby version from another `bzl` file, and just modify that. eg `//rb:ruby_version.bzl` could declare a single value (`rb_version`), and then we just rewrite that file in the build matrix